extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/linkHandler/YoutubeParsingHelper.java

```diff
@@ -398,14 +398,34 @@ public static JsonArray getJsonResponse(String url, Localization localization) t
      * @throws ContentNotAvailableException if an alert is detected
      */
     public static void defaultAlertsCheck(JsonObject initialData) throws ContentNotAvailableException {
```

Comment:
I suggest to use the newly introduced default values, that way, we can avoid all those nasty try-catch (and null pointer exceptions).

###### Maybe we should switch to a new library that does this by default, or even fork and modify the one we are using.

```java
final JsonObject alertRenderer = initialData
        .getArray("alerts", EMPTY_ARRAY)
        .getObject(0, EMPTY_OBJECT)
        .getObject("alertRenderer", EMPTY_OBJECT);

final String alertType = alertRenderer.getString("type", EMPTY_STRING);
if (alertType.equalsIgnoreCase("ERROR")) {
    final String alertText = alertRenderer
            .getObject("text", EMPTY_OBJECT)
            .getString("simpleText", EMPTY_STRING);

    if (alertText.isEmpty()) {
        throw new ContentNotAvailableException("Got an unknown alert error");
    } else {
        throw new ContentNotAvailableException("Got alert error: \"" + alertText + "\"");
    }
}
```